packages/grafana-ui/src/components/Forms/Checkbox.story.tsx

```diff
@@ -80,6 +81,7 @@ InAField.args = {
     'Annotation queries can be toggled on or of at the top of the dashboard. With this option checked this toggle will be hidden.',
   disabled: false,
   indeterminate: false,
+  invalid: false,
 };
 
 export const AllStates: ComponentStory<typeof Checkbox> = (args) => {
```

Comment:
Since the story is supposed to make all available states visible I would suggest to add the invalid state to it. What do you think?

Reply:
Thank you for your feedback! I wouldn't realise to add it as an example if you hadn't told me 🤗